openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/SYS2/activity_eligibility/SYS2_eligibility.py

```diff
@@ -66,19 +44,19 @@ def formula(buildings, period, parameters):
             False,
             True
         ])
-        #single speed is YES and single speed input power is YES or multiple speed is YES and multiple speed input power is YES
+
         speed_and_input_power_eligible = np.select([
-            (np.logical_not(pump_multiple_speed) * (input_power == SYS2_input_power_Options.single_speed)),
-            (np.logical_not(pump_multiple_speed) * (input_power != SYS2_input_power_Options.single_speed)),
-            (pump_multiple_speed * (input_power == SYS2_input_power_Options.multiple_speed)),
-            (pump_multiple_speed * (input_power != SYS2_input_power_Options.multiple_speed))
+            (np.logical_not(pump_multiple_speed) * single_speed_input_power),
```

Comment:
can you review if these conditions look ok?  @k8mckay 

(np.logical_not(pump_multiple_speed) * single_speed_input_power) -> True
(np.logical_not(pump_multiple_speed) * np.logical_not(single_speed_input_power)) -> False
(pump_multiple_speed * multiple_speeds_input_power) -> True
(pump_multiple_speed * np.logical_not(multiple_speeds_input_power) ) -> False